# SETUP

global_frame: /map
robot_base_frame: /amigo/base_link
rolling_window: false
track_unknown_space: false
resolution: 0.05

update_frequency: 1.0
publish_frequency: 1.0

always_send_full_costmap: true

# LAYER SETUP

plugins: 
    # Obstacles
    - {name: ed_occupancy_grid,                 type: "costmap_2d::StaticLayer"}
    - {name: robot_footprint,                   type: "costmap_2d::FootprintLayer"}
    - {name: configuration_space,               type: "costmap_2d::InflationLayer"}

# Ed World model layer
ed_occupancy_grid:
    map_topic: /ed_map
    track_unknown_space: false

# Workspace to configuration space 
configuration_space:
    use_footprint: false
    target_cell_value: 254
    dilation_cell_value: 253
    dilation_radius: 0.37 # 2cm margin
    inflation_radius: 0.37 
